### DISCUSSION TITLE
cannot import name 'kie_predictor' from 'doctr.models'

### DISCUSSION THREAD

**Discussion by entiolliko** [mindee/doctr#1216] [Q&A]
I just installed DocTR and I am interested in the KIE detector as shown in the README of the github repo. Unfortunatelly I couldn't find anything on the installation of the detector on the website. I wanted to ask how can I do it.
Thanks

**Comment by felixT2K** [comment]
Hi @entiolliko You need to clone the repo to install the dev version. :)
```
git clone https://github.com/mindee/doctr.git
# Install your prefered backend (pytorch or tensorflow) if not already installed then:
cd doctr
pip3 install -e .
```
